src/spi_comms.c

```diff
@@ -128,7 +128,8 @@ void opt_transfer_reading(uint8_t well_info){
 
         shift = shift - 8;
 
-        // might need small delay here, will test
+        // small delay to give time for DATA_RDYn to go high
+        _delay_us(10);

```

Comment:
Don't put the delay here, I meant a delay on SSM so it waits for optical to change DATA_RDYn